elementor/elementor
# Tweak: Add `avif` support in lightbox-manager [ED-18470]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* adding avif support in the lightbox (e.g. gallery widget)

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

* add a gallery widget with avif files and click the small image
* lightbox should open

## Quality assurance


Fixes https://github.com/orgs/elementor/discussions/25175
